tools/BUILD

```diff
@@ -0,0 +1,45 @@
+# *******************************************************************************
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+
+package(default_visibility = ["//visibility:public"])
+
+exports_files([
```

Comment:
it's currently quite a mess in the directory, maybe for the future these could have a prefix or a subdirectory?